tasks/symlink-shared.yml

```diff
@@ -13,3 +13,19 @@
     path: "{{ ansistrano_release_path.stdout }}/{{ item }}"
     src: "{{ item | regex_replace('[^\\/]*', '..') }}/../shared/{{ item }}"
   with_items: "{{ ansistrano_shared_paths }}"
+
+# Remove previous .rsync-filter file (rsync current deployment)
```

Comment:
Not sure about this, what happens if the deployment strategy is rsync and there is a .rsync-filter file in your repo (like I do for instance sometimes)